synapse/tests/test_swarm_runtime.py

```diff
@@ -290,3 +290,15 @@ def test_swarm_runtime_has(self):
         self.assertEqual( tufos[0][1].get('foo:bar'), 'baz' )
 
         env.fini()
+
+    def test_swarm_runtime_range(self):
+
+        env = self.getSwarmEnv()
+        answ = env.runt.ask('foo:bar*range=("cat","goat")')
+
+        tufos = answ.get('data')
+
+        self.assertEqual( len(tufos), 1 )
+        self.assertEqual( tufos[0][1].get('foo:bar'), 'faz' )
+
+        env.fini()
```

Comment:
Looks like we may still need a couple test cases:
1. filter by "range"
2. lift by "in"
3. filter by "in"